go.sum

```diff
@@ -1,5 +1,5 @@
-github.com/cenkalti/backoff/v4 v4.1.3 h1:cFAlzYUlVYDysBEH2T5hyJZMh3+5+WCBvSnK6Q8UtC4=
-github.com/cenkalti/backoff/v4 v4.1.3/go.mod h1:scbssz8iZGpm3xbr14ovlUdkxfGXNInqkPWOWmG2CLw=
+github.com/ItalyPaleAle/backoff/v4 v4.2.0 h1:bcc9j45CtY0SURvcuoiGK2L5rq9ZTtfwUxUgwyZrLGw=
+github.com/ItalyPaleAle/backoff/v4 v4.2.0/go.mod h1:Y3VNntkOUPxTVeUxJ/G5vcM//AlwfmyYozVcomhLiZE=
```

Comment:
Ok, but hopefully you will continue to maintain this.

Reply:
I will give the maintainer a few days to respond.

If they accept the PR (which I'm not super confident about, given other comments that make it sound like they consider the library "done"), I'll remove the replace.

If they don't, two options:

- We can vendor the code in dapr/kit
- We can fork it as dapr/backoff